src/core/components/config.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const promisify = require('promisify-es6')
+const defaultConfig = require('../runtime/config-nodejs.js')()
```

Comment:
Please call this funciton every time you need `defaultConfig` because otherwise it is global, shared and can be mutated.

Reply:
Done